sharktank/tests/ops/sharded_test.py

```diff
@@ -1366,6 +1366,55 @@ def testGroupNormSplitChannelSharding(self):
         assert expected_bias.is_deep_equal(sharded_theta("bias"), compare_name=False)
 
 
+class Scatter_Test(unittest.TestCase):
+    def setUp(self):
+        import numpy as np
+
+        np.random.seed(12345)
+        self.rng = np.random.default_rng()
```

Comment:
You would need to seed the RNG to avoid a non-deterministic test?
I think we need to refactor at some point a introduce a base test case class that does all this by RNG seeding default.